### PR TITLE
Added disabled input property to radio group component

### PR DESCRIPTION
**Pull request by Blackbaud-AlexKingman**
Addresses #80 

New property on `SkyRadioGroupComponent`:
```
@Input() disabled
Indicates whether to disable the input. This property accepts `boolean` values.
```

Note: Also added same description to already-existing `disabled` property on the `SkyRadioComponent`.

### CODE REVIEW

File: src/app/public/modules/radio/fixtures/radio-group-reactive.component.fixture.html [hunk 1]
```diff
@@ -1,10 +1,10 @@
 <form
   [formGroup]="radioForm"
 >
-  <sky-radio-group *ngIf="radioGroupEnabled"
+  <sky-radio-group *ngIf="showRadioGroup"
     class="form-group radio"
-    formControlName="option"
-    name="option"
+    formControlName="radioGroup"
+    name="radioGroup"
```
Comment (Blackbaud-AlexKingman) [inline src/app/public/modules/radio/fixtures/radio-group-reactive.component.fixture.html]: ☝️ updated the control name and element name to be more semantic. This isn't really an `option`, but instead the entire radio group.

File: src/app/public/modules/radio/fixtures/radio-group-reactive.component.fixture.ts [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+} from '@angular/forms';
+
+import {
+  SkyRadioGroupComponent
+} from '../radio-group.component';
+
+@Component({
+  templateUrl: './radio-group-reactive.component.fixture.html'
+})
+export class SkyRadioGroupReactiveFixtureComponent {
```
Comment (Blackbaud-AlexKingman) [inline src/app/public/modules/radio/fixtures/radio-group-reactive.component.fixture.ts]: ☝️ What's going on with these fixtures? Don't panic - `SkyRadioGroupReactiveFixtureComponent` appears to be a new file in this diff, but its actually the old fixture that was there before. I just renamed it to make way for a template-driven fixture. All our coverage was only using reactive forms. I created a new template-driven fixture to start covering TD forms. See `SkyRadioGroupFixtureComponent` below.

File: src/app/public/modules/radio/radio-group.component.spec.ts [hunk 3]
```diff
@@ -17,29 +17,34 @@ import {
...
-describe('Radio group component', function () {
-  let fixture: ComponentFixture<SkyRadioGroupTestComponent>;
-  let componentInstance: SkyRadioGroupTestComponent;
+//#region helpers
+function getRadios(radioFixture: ComponentFixture<any>): NodeListOf<HTMLInputElement> {
+  return radioFixture.nativeElement.querySelectorAll('.sky-radio-input');
+}
 
-  //#region helpers
-  function getRadios(radioFixture: ComponentFixture<any>) {
```
Comment (Blackbaud-AlexKingman) [inline src/app/public/modules/radio/radio-group.component.spec.ts]: A lot of our helpers weren't typecast and not available in a global context. I moved these outside the `describe` block so I could use them between different fixtures.

File: src/app/public/modules/radio/radio-group.component.spec.ts [hunk 4]
```diff
@@ -115,35 +120,14 @@ describe('Radio group component', function () {
...
-      option: componentInstance.options[1]
+      radioGroup: componentInstance.options[1]
     });
     fixture.detectChanges();
     tick();
 
     expect(componentInstance.radioGroupComponent.value.name).toEqual('Harima Kenji');
   }));
 
-  it('should handle disabled state properly', fakeAsync(function (done: Function) {
```
Comment (Blackbaud-AlexKingman) [inline src/app/public/modules/radio/radio-group.component.spec.ts]: This test was covering radio component logic and NOT radio group logic. I double checked the `radio.component.spec.ts` file and saw we have a duplicate test there. Coverage did not drop either. Removed!

File: src/app/public/modules/radio/radio-group.component.ts [hunk 5]
```diff
@@ -154,6 +172,15 @@ export class SkyRadioGroupComponent implements AfterContentInit, AfterViewInit,
     this.value = value;
   }
 
+  /**
+   * @internal
+   * Sets the disabled state of the control. Implemented as a part of ControlValueAccessor.
+   * @param isDisabled Whether the control should be disabled.
```
Comment (blackbaud-johnly) [inline src/app/public/modules/radio/radio-group.component.ts]: Not sure if this line is going to show up anywhere, but if so, I'd change this slightly to follow the pattern in the docs: `Indicates whether to disable the control.`

File: src/app/public/modules/radio/radio.component.ts [hunk 6]
```diff
@@ -185,6 +203,15 @@ export class SkyRadioComponent implements OnDestroy, ControlValueAccessor {
     this.changeDetector.markForCheck();
   }
 
+  /**
+   * @internal
+   * Sets the disabled state of the control. Implemented as a part of ControlValueAccessor.
+   * @param isDisabled Whether the control should be disabled.
```
Comment (blackbaud-johnly) [inline src/app/public/modules/radio/radio.component.ts]: Not sure if this line is going to show up anywhere, but if so, I'd change this slightly to follow the pattern in the docs: `Indicates whether to disable the control.`

File: src/app/public/modules/radio/radio-group.component.ts (hunk 5 above)
Comment (Blackbaud-AlexKingman) [reply src/app/public/modules/radio/radio-group.component.ts]: Done.

File: src/app/public/modules/radio/radio.component.ts (hunk 6 above)
Comment (Blackbaud-AlexKingman) [reply src/app/public/modules/radio/radio.component.ts]: Done.

**Pull request merged**